docs/installation.rst

```diff
@@ -96,6 +96,34 @@ Setup
 
   If you have turned on user takeover setting define URL where will be used after user account takeover. Default value is ``'/'``.
 
+.. attribute:: AUTH_TOKEN_TWO_FACTOR_ENABLED
+
+  Two factor authentication is enabled or disabled. By default value ``False`` is set.
+
+.. attribute:: AUTH_TOKEN_TWO_FACTOR_REDIRECT_URL
+
+  The path the user is redirected to after successful two factor authentication. Default value ``'login-code-verification/'``.
+
+.. attribute:: AUTH_TOKEN_TWO_FACTOR_AUTHORIZATION_SLUG
+
+  Authorization request slug for two factor authorization. Default value is ``'2FA'``.
+
+.. attribute:: AUTH_TOKEN_TWO_FACTOR_CODE_GENERATING_FUNCTION
+
+  Function which generates code for two factor authentication. Default value is ``'auth_token.utils.generate_two_factor_key'``.
+
+.. attribute:: AUTH_TOKEN_TWO_FACTOR_CODE_GENERATING_FUNCTION_LENGTH
+
+  Default length of two factor code. Default value is ``5``.
```

Comment:
Length of 6 is much more common (all services I use do have a 6 character code), I don't know the reason, just saying.